hack/minikube-rook-setup.sh

```diff
@@ -99,10 +99,38 @@ fi
 minikube ssh "sudo mkdir -p /mnt/vda1/rook/ && sudo ln -sf /mnt/vda1/rook/ /var/lib/rook" --profile="${PROFILE}"
 
 ## Create and attach an OSD disk for Ceph ##
-if [[ $(virsh pool-list | grep -wc "${POOL_NAME}") == 0 ]]; then
+pool=$(virsh pool-dumpxml $POOL_NAME)
+pool_target=${pool#*<target>}
+pool_target=${pool_target%</target>*}
+pool_target_path=${pool_target#*<path>}
+pool_target_path=${pool_target_path%</path>*}
+pool_target_path_set()
+{
+	echo "${pool%<target>*}"\<target\>"${pool_target%<path>*}"\<path\>"$IMAGE_DIR"\</path\>"${pool_target#*</path>}"\</target\>"${pool#*</target>}" >/tmp/$$
+	virsh pool-define /tmp/$$
+	rm -f /tmp/$$
+}
+pool_target_path_set_unqualified()
+{
+	test 'Pool '$POOL_NAME' XML configuration edited.' = "$(\
+	EDITOR=sed\ -i\ \''s,<path>.*</path>,<path>'$IMAGE_DIR'</path>,'\' virsh pool-edit $POOL_NAME \
+	)"
+}
```

Comment:
There may be some advantages to [pool-edit](https://libvirt.org/manpages/virsh.html#pool-edit) , but qualifying the path as a target was not straightforward to me as `sed` processing unit is typically a line and target and path tags span at least two lines.